assets/src/js/header.js

```diff
@@ -227,5 +227,44 @@ export const setupHeader = () => {
     }
   });
 
+  /**
+   * Set the submenu behavior.
+   * Desktop should render drop/down list
+   * Mobile should render an accordion list
+   */
+  const setupSubmenu = () => {
+    document.querySelectorAll('.nav-submenu').forEach(submenu => {
+      // Find its parent menu item
+      const menuItem = document.querySelector(`*[data-menu="${submenu.dataset.parentMenu}"]`);
+      const list = submenu.querySelector('ul');
+
+      // The limits should be 6 for a scrollable list
+      if(list.children.length >= 6) {
```

Comment:
I see that you actually already know the `max-height` it could be, I think if you just set [this value](https://github.com/greenpeace/planet4-master-theme/pull/1604/files#diff-91009267fa7864d1b148d657a615793b252f3e148c4001b639eab5a4c51324e4R228) as max-height to all of them with the auto overflow-y then it should do the same behaviour without the need for this extra JS and class 🤔